src/modules/hub.bicep

```diff
@@ -7,7 +7,9 @@ param hubName string
 
 // Generate unique storage account name
 var storageAccountSuffix = 'store'
+var dataFactorySuffix = 'factory'
 var storageAccountName = '${substring(replace(toLower(hubName), '-', ''), 0, 24 - length(storageAccountSuffix))}${storageAccountSuffix}'
+var dataFactoryName = '${substring(replace(toLower(hubName), '-', ''), 0, 24 - length(dataFactorySuffix))}${dataFactorySuffix}'
```

Comment:
Does ADF have the same lowercase, no special characters limits as storage? If not, can we keep the original casing and not remove characters?

Reply:
fixed

Reply:
removed